src/widgetresize/resizer.js

```diff
@@ -269,6 +277,12 @@ export default class Resizer {
 	_cleanup() {
 		this._sizeUI.dismiss();
 		this._sizeUI.isVisible = false;
+
+		const editingView = this._options.editor.editing.view;
+
+		editingView.change( writer => {
```

Comment:
I wonder if this is the reason of:

![](https://user-images.githubusercontent.com/247363/79739606-b3ea4400-82fe-11ea-885f-098a6e6666dd.gif)

Found on Firefox but also visible on Chrome: after ending resizing the image border of the original size flickers. It happens on every drag (not GIF drops some frames and is not always visible).

What I can thinkof is a single commit block (if doable).

Reply:
Yeah, this is probably because things are done in two batches and the browser gets a chance to re-pain in between. I don't know how to address this unless I spend some serious time digging into the code.

Reply:
Doh :facepalm: @jodator is right. I was a little overconfident with the change and did it only by writing unit test and making a fix. Without realizing there might be some rendering artifacts :smiley: